include/nbl/video/SPhysicalDeviceLimits.h

```diff
@@ -0,0 +1,497 @@
+
+    /* DescriptorIndexingPropertiesEXT ---> MOVED TO Vulkan 1.2 Core  *//* provided by VK_AMD_shader_core_properties */
+
+    /* DiscardRectanglePropertiesEXT *//* provided by VK_EXT_discard_rectangles */
+    //uint32_t           maxDiscardRectangles;
+            
+    /* ExternalMemoryHostPropertiesEXT *//* provided by VK_EXT_external_memory_host */
+    //VkDeviceSize       minImportedHostPointerAlignment;
+    
+    /* FragmentDensityMapPropertiesEXT *//* provided by VK_EXT_fragment_density_map */
+    //VkExtent2D         minFragmentDensityTexelSize;
+    //VkExtent2D         maxFragmentDensityTexelSize;
+    //VkBool32           fragmentDensityInvocations;
+    
+    /* FragmentDensityMap2PropertiesEXT *//* provided by VK_EXT_fragment_density_map2 */
+    //VkBool32           subsampledLoads;
+    //VkBool32           subsampledCoarseReconstructionEarlyAccess;
+    //uint32_t           maxSubsampledArrayLayers;
+    //uint32_t           maxDescriptorSetSubsampledSamplers;
+    
+    /* GraphicsPipelineLibraryPropertiesEXT *//* provided by VK_EXT_graphics_pipeline_library */
+    //VkBool32           graphicsPipelineLibraryFastLinking;
+    //VkBool32           graphicsPipelineLibraryIndependentInterpolationDecoration;
+
+    /* InlineUniformBlockPropertiesEXT ---> MOVED TO Vulkan 1.3 Core  */
+
+    /* LineRasterizationPropertiesEXT *//* provided by VK_EXT_line_rasterization */
+    //uint32_t           lineSubPixelPrecisionBits;
+
+    /* MultiDrawPropertiesEXT *//* provided by VK_EXT_multi_draw */
+    //uint32_t           maxMultiDrawCount;
+
+    /* PCIBusInfoPropertiesEXT *//* provided by VK_EXT_pci_bus_info */
+    //uint32_t           pciDomain;
+    //uint32_t           pciBus;
+    //uint32_t           pciDevice;
+    //uint32_t           pciFunction;
+
+    /* DrmPropertiesEXT *//* provided by VK_EXT_physical_device_drm */
+    //VkBool32           hasPrimary;
+    //VkBool32           hasRender;
+    //int64_t            primaryMajor;
+    //int64_t            primaryMinor;
+    //int64_t            renderMajor;
+    //int64_t            renderMinor;
+
+    /* ProvokingVertexPropertiesEXT *//* provided by VK_EXT_provoking_vertex */
+    //VkBool32           provokingVertexModePerPipeline;
+    //VkBool32           transformFeedbackPreservesTriangleFanProvokingVertex;
+
+    /* Robustness2PropertiesEXT *//* provided by VK_EXT_robustness2 */
+    //VkDeviceSize       robustStorageBufferAccessSizeAlignment;
+    //VkDeviceSize       robustUniformBufferAccessSizeAlignment;
+
+    /* SamplerFilterMinmaxPropertiesEXT ---> MOVED TO Vulkan 1.2 Core  */
+
+    /* SampleLocationsPropertiesEXT *//* provided by VK_EXT_sample_locations */
+    //VkSampleCountFlags    sampleLocationSampleCounts;
+    //VkExtent2D            maxSampleLocationGridSize;
+    //float                 sampleLocationCoordinateRange[2];
+    //uint32_t              sampleLocationSubPixelBits;
+    //VkBool32              variableSampleLocations;
```

Comment:
`sampleLocationSampleCounts!=0` and `variableSampleLocations=true` only if
https://www.khronos.org/registry/OpenGL/extensions/ARB/ARB_sample_locations.txt
or 
https://www.khronos.org/registry/OpenGL/extensions/AMD/AMD_sample_positions.txt
available

otherwise `sampleLocationSampleCounts=0` and `variableSampleLocations=false`


you can always get the remaining info from:
https://www.khronos.org/registry/OpenGL/extensions/NV/ARB_texture_multisample.txt
which I think is core in GL 4.4 and possibly even in GLES 3.1